Kernel/Devices/HID/PS2MouseDevice.cpp

```diff
@@ -97,6 +98,14 @@ MousePacket PS2MouseDevice::parse_data_packet(const RawPacket& raw_packet)
         // -1 in 4 bits
         if (z == 15)
             z = -1;
+
+        if ((raw_packet.bytes[3] & 0xc0) == 0x40) {
+            // FIXME: obviously I've messed up signs somehwere
```

Comment:
Typo; also, please rephrase this to not refer to "you" specifically. Once the code is in, it's everyone's code and everyone may fix the issue.